### ISSUE TITLE
highlight customizer comments

### ISSUE THREAD

**Issue by yannickbattail** [ldenisey/idea-openscad#39]
highlight in a different color comments that are Customizer parameter

code after `//` in the same line of variable initialization and tab name between `/* [] */` should be in another color

example of code
```openscad
string_enumeration = "value2"; // [value1, value2, value3]
string_enumeration_with_description = "value3"; // [value1:description1 , value2:description2, value3:description3]
/* [TAB_NAME] */
number_with_min_max = 1; // [0.5:3]
number_with_min_maxand_step = 1; // [0.5:0.1:3]
string_with_length="hello"; //10
```

[Customizer doc](https://en.wikibooks.org/wiki/OpenSCAD_User_Manual/Customizer)

see example with [vsCode extension](https://github.com/Antyos/vscode-openscad)
![image](https://user-images.githubusercontent.com/1282175/187401887-35f896fd-f6e8-4014-96c7-c904952dc678.png)

**Comment by ldenisey** [comment]
Added in #51.

**Comment by ldenisey** [comment]
Will be released in version 2.4.0 coming soon.

**Comment by yannickbattail** [comment]
great! ❤️
